### PR TITLE
Id feature accountform

### LINKED ISSUE #5: Create AccountForm UI component
- Create account form  UI

### PR DESCRIPTION
**Pull request by IED92**
Created form components and Home page

### CODE REVIEW

File: client/js/components/LoginForm/LoginForm.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+const LoginForm = () => {
+  const [email, setEmail] = React.useState('');
+  const [password, setPassword] = React.useState('');
+
+  //   const {signIn} = React.useContext(AuthContext);
```
Comment (siddharthparmar7) [inline client/js/components/LoginForm/LoginForm.js]: remove commented code if not needed.

File: client/js/components/SignUpForm/SignUp.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+const SignUpForm = () => {
+  const [email, setEmail] = React.useState('');
+  const [password, setPassword] = React.useState('');
+
+  //   const {signUp} = React.useContext(AuthContext);
```
Comment (siddharthparmar7) [inline client/js/components/SignUpForm/SignUp.js]: same as above.

File: client/js/components/SignUpForm/SignUp.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+      <TextInput placeholder="email" value={email} onChangeText={setEmail} />
+      <TextInput
+        placeholder="Password"
+        value={password}
+        onChangeText={setPassword}
+        secureTextEntry
+      />
+      <Button
+        title="Sign Up"
+        onPress={() => signUp({firstName, lastName, email, password})}
```
Comment (siddharthparmar7) [inline client/js/components/SignUpForm/SignUp.js]: there is no method called signUp in your code. Please define signUp. Make sure when we merge your code it doesn't throw any errors.

File: client/js/components/LoginForm/LoginForm.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+  return (
+    <View>
+      <TextInput placeholder="email" value={email} onChangeText={setEmail} />
+      <TextInput
+        placeholder="Password"
+        value={password}
+        onChangeText={setPassword}
+        secureTextEntry
+      />
+      <Button title="Login" onPress={() => signIn({email, password})} />
```
Comment (siddharthparmar7) [inline client/js/components/LoginForm/LoginForm.js]: signIn method is missing. Please add it.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Please make changes to your code so that it doesn't throw errors. Make sure when you submit a pull request for review it is a complete feature. Thank you.

**Review by KieranDevlin — DISMISSED** [review]

**Comment by siddharthparmar7** [comment]
Please resolve all the merge conflicts.

**Review by KieranDevlin — APPROVED** [review]
Merge conflicts corrected

**Review by siddharthparmar7 — APPROVED** [review]

**Pull request merged**